robottelo/ui/base.py

```diff
@@ -175,6 +160,21 @@ def delete_entity(self, name, really, name_locator, del_locator,
         strategy, value = del_locator
         self.click((strategy, value % name), wait_for_ajax=False)
         self.handle_alert(really)
+        for _ in range(3):
```

Comment:
Was not able to get that it is one second of wait. Thanks for explaining.

Maybe this worth a comment explaining the approach?